src/pretix/control/views/organizer.py

```diff
@@ -2311,6 +2311,14 @@ def get_context_data(self, **kwargs):
             o.icnt = annotated.get(o.pk)['icnt']
             o.sales_channel_obj = scs[o.sales_channel]
 
+        ctx["lifetime_spending"] = (
+            self.get_queryset()
+            .filter(status__in=[Order.STATUS_PAID, Order.STATUS_PENDING])
```

Comment:
This could either include pending orders or not, depending on expected behavior. Would someone want to see spending that is yet to come, but will probably soon be paid, or would it be better to show only the numbers for confirmed paid orders?

Reply:
There's really no good way to determine this. Some of our customers would expect one, some the other, but I really don't want a switch and my vote goes for "just the paid ones".

Reply:
We could add a ``data-toggle="tooltip" title="{% trans "This includes all paid orders by this customer across all your events." %}"`` to at least make it understandable